### DISCUSSION TITLE
Particle editors for Axmol

### DISCUSSION THREAD

**Discussion by j-jorge** [axmolengine/axmol#2198] [Q&A]
What are the options to design particle systems to be used within Axmol?

In Cocos2d-x times, particles could be imported from external tools (e.g. Particle Designer) and if I recall correctly there was an online editor like EffectHub but without the need to register. I wonder if there is something nice for Axmol today, ideally open source, and that definitely do not ask to register to anything.

Since there is everything in Axmol to display the particle systems, and since there is also an integration of Dear ImGui to create UIs, maybe there should be a particle editor integrated in Axmol?

**Comment by aismann** [comment]
@j-jorge 
Have you tried extension Effekseer?
There should be a free editor too.

@j-jorge
plz set answer flag too. Thanks.

**Comment by iAndyHD3** [comment]
It would be a good idea to add some editing options into the inspector, but ofcourse that needs to be added by someone
